__tests__/app.test.js

```diff
@@ -38,3 +38,42 @@ describe('GET /api/categories', () => {
     });
 });
 
+describe('GET /api/reviews/:review_id', () => {
+  test("Get status 200 response", () => {
+    return request(app).get("/api/reviews/1").expect(200);
+  });
+  test("Get review object corresponding to review_id", () => {
+    return request(app)
+      .get("/api/reviews/1")
+      .expect(200)
+      .then((response) => {
+          expect(typeof response.body.review.review_id).toBe("number");
```

Comment:
this review_id should be more specific, it could be any review and it would pass, which one are you looking at? 1.